package.json

```diff
@@ -31,6 +30,6 @@
   },
   "dependencies": {
     "@types/react": "^15.0.21",
```

Comment:
to summary:
in https://github.com/Shopify/neutron/blob/master/package.json we have all the type files in `devDependencies` because we are not exporting it anywhere and the type checking only run in development.

But it is because `javascript-utilities` are expose to its consumer and has type as part of the signature that it needed to be a `dependencies `